htdocs/core/boxes/box_project.php

```diff
@@ -99,6 +100,7 @@ public function loadBox($max = 5)
             $sql = "SELECT p.rowid, p.ref, p.title, p.fk_statut, p.public";
             $sql.= " FROM ".MAIN_DB_PREFIX."projet as p";
             $sql.= " WHERE p.fk_statut = 1"; // Only open projects
+			$sql .= " AND entity IN (".getEntity('projet').")"; // Only current entity or severals if permission ok
```

Comment:
@mapiolca 

better is : 
use "project" instead "projet" and use entity filter before other filters

```
$sql .= " WHERE entity IN (".getEntity('project').")"; // Only current entity or severals if permission ok
$sql.= " AND p.fk_statut = 1"; // Only open projects
```

Reply:
Done